src/styles/blocks/snake.scss

```
@@ -0,0 +1,59 @@
+.snake {
+  display: grid;
+
+ &__block {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    height: 300px;
+    font-size: 100px;
+    font-family: Arial, Helvetica, sans-serif;
+    color: #fff;
+
+    @for $index from 1 through 6 {
+      &:nth-child(#{$index})::before {
+       content: "#{$index}";
+      }
+    }
+
+    &--1 {
+      background-color: rgb(255,2,0);
+    }
+
+    &--2 {
+      background-color: rgb(204,2,0);
+    }
+
+    &--3 {
+      background-color: rgb(153,0,1);
+    }
+
+    &--4 {
+      background-color: rgb(102,0,0);
+    }
+
+    &--5 {
+      background-color: rgb(51,1,0);
+    }
+
+    &--6 {
+      background-color: rgb(0,0,0);
+    }
+  }
+    @media (min-width: 300px) and (max-width: 600px) {
+      grid-template-columns: $block-width;
+    }
+
+    @media (min-width: 600px) and (max-width: 900px) {
```

```suggestion
    @media (min-width: 600px) {
```

no need to set the highest value, the next media query will override it 😉

---

Agree with Vitaly